sdk/core/core-client-rest/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Allow `number` path parameters.
```

Comment:
Add a link so that people reading CHANGELOG can jump to PR or issue easily?

Reply:
```suggestion
- Allow `number` path parameters. PR [#31352](https://github.com/Azure/azure-sdk-for-js/pull/31352/files)
```